Increase the memory sizes for LaTeX

These increased stack sizes should be sufficient for LaTeX to handle the
huge index of the Sage PDF reference manual.

diff -ru a/sphinx/texinputs/Makefile b/sphinx/texinputs/Makefile
--- a/sphinx/texinputs/Makefile	2016-03-06 06:25:30.000000000 +0100
+++ b/sphinx/texinputs/Makefile	2016-04-06 19:33:07.248103397 +0200
@@ -11,8 +11,10 @@
 # format: pdf or dvi
 FMT = pdf
 
-LATEX = latex
-PDFLATEX = pdflatex
+LATEXENV = env pool_size=4000000 save_size=50000 extra_mem_top=2000000
+
+LATEX = $(LATEXENV) latex
+PDFLATEX = $(LATEXENV) pdflatex
 MAKEINDEX = makeindex
 
 all: $(ALLPDF)
